app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
+from app.players.player import Player

```

Comment:
Here too, you can import ABC from Player:
`from app.players.player import Player, ABC`
don`t need the first import